orttraining/tools/ci_test/results/ci-mi100.huggingface.gpt2-rocm4.3.1.json

```diff
@@ -53,5 +53,5 @@
             "loss": 1.5503
         }
     ],
-    "samples_per_second": 25.587
+    "samples_per_second": 24.756
```

Comment:
>24.756 [](http://example.com/codeflow?start=26&length=6)

intresting, how do you choose this magic number?

Reply:
I did some experiments and compute a average number of them. If there is a better way I can try again.